position/controllers.py

```diff
@@ -226,6 +226,87 @@ def return_most_likely_data(from_position, to_position, attribute):
     return getattr(to_position, attribute)
 
 
+def move_positions_to_another_candidate(from_candidate_id, from_candidate_we_vote_id,
+                                        to_candidate_id, to_candidate_we_vote_id,
+                                        public_or_private):
+    status = ''
+    success = True
+    position_entries_moved = 0
+    position_entries_not_moved = 0
+    position_list_manager = PositionListManager()
+
+    retrieve_public_positions = public_or_private
```

Comment:
This variable naming convention "retrieve_public_positions" is no longer valid and can be updated.